README.md

```diff
@@ -82,6 +82,8 @@ Even with these basic mechanics, there is quite a lot of complexity in creating
     * OSX: either [Virtualbox](https://www.virtualbox.org/wiki/Downloads) or [VMWare Fusion](http://www.vmware.com/products/fusion.html).
 * Download Docker, Minikube, and Kubectl before running the script.
     * On Mac ([download homebrew](https://brew.sh/)) and run `brew update && brew install kubectl && brew cask install docker minikube virtualbox`.
+    * On Ubuntu ([download snap](https://snapcraft.io/)) and run `sudo snap install kubectl --classic` and follow the ([docker installation instructions](https://docs.docker.com/install/linux/docker-ce/ubuntu/)).
```

Comment:
and run, and follow... I would write then follow, or replace first and with a comma?